### DISCUSSION TITLE
Rowy - Firebase reads

### DISCUSSION THREAD

**Discussion by tinrunje** [AntlerVC/firetable#1142] [Support / Q&A] [answered]
I have noticed that once connecting my Firebase to Rowy, the amount of reads in my FB backend has significantly increased. Is there some documentation of how this works and if this can be limited? 
I understand Rowy reads this data in real-time, however if the costs are X times higher, it might cause problems...

**Comment by harinij** [comment, marked as answer]
Hi @tinrunje, Rowy only reads the data that is rendered on the screen and should not result in any unnecessary reads/excess cost. You can read more about it here on our [Docs FAQ](https://docs.rowy.io/faqs/database#how-does-rowy-perform-for-a-large-number-of-rows-how-many-rows-you-can-render-in-one-session-is-there-any-throttling)

However, we had a bug on the sorting feature released 2 days back which was causing a [bug](https://github.com/rowyio/rowy/issues/1137), this has been resolved today - so you should not be seeing this issue anymore if this is something you faced from the weekend. Thanks for your patience.
